ui/shared/components/panel/acmg/AcmgCriteria.jsx

```diff
@@ -118,7 +144,8 @@ class AcmgCriteria extends React.PureComponent {
       const elements = rows.map((row, rowIdx) => {
         const criteriaUsed = this.getCriteriaUsed()
         return (
-          <Table.Row>
+          // eslint-disable-next-line react/no-array-index-key
+          <Table.Row key={rowIdx}>
```

Comment:
It seems that the options defined in `ACMG_DROP_DOWN_OPTIONS` have a `key` field. Can we use it for the key?

Reply:
ideally yes, but they don't. And adding it that would involve me reading through about 100 different constants that I didn't write and am not familiar with, reconfiguring lists of lists to be lists of objects with list keys and then updating the rendering code accordingly, and then figuring out how to add meaningful keys. If you or I had implemented this feature I absolutely would have wanted to do that. But given that this is an external contribution the point was just to get this working without spamming the console with errors, and maybe some day in the future (probably when it needs to be updated) I'll clean it up